### PR TITLE
Payment component and Stripe integration

### PR DESCRIPTION
**Pull request by migle-kirilovaite**

### CODE REVIEW

**Review by d-pilipavicius — DISMISSED** [review]
LGTM

**Review by EvelinaTemnolonskyte — DISMISSED** [review]
LGTM

File: src/main/java/com/example/demo/payments/API/Endpoints/PaymentController.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    @PostMapping("/complete")
+    public ResponseEntity<?> completeOrderPayment(@RequestParam UUID orderId) {
+        boolean success = paymentApplicationService.completeOrderPayment(orderId);
+        if (success) {
+            return ResponseEntity.ok().build();
+        } else {
+            return ResponseEntity.status(400).body("Failed to close the order.");
+        }
+    }
+}
```
Comment (d-pilipavicius) [inline src/main/java/com/example/demo/payments/API/Endpoints/PaymentController.java]: New rule for auth, "/{businessId}" must be added to the path, check: https://github.com/d-pilipavicius/staciakampiai/pull/12

File: src/main/java/com/example/demo/payments/API/Endpoints/PaymentWebhookController.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
+package com.example.demo.payments.API.Endpoints;
+
+import com.example.demo.payments.API.DTOs.CheckoutSessionCompletedDTO;
+import com.example.demo.payments.Domain.Services.PaymentService;
+import lombok.AllArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/v1/payments/webhooks")
```
Comment (d-pilipavicius) [inline src/main/java/com/example/demo/payments/API/Endpoints/PaymentWebhookController.java]: https://github.com/d-pilipavicius/staciakampiai/pull/12 most likely here too

**Review by d-pilipavicius — APPROVED** [review]
LGTM

**Review by EvelinaTemnolonskyte — APPROVED** [review]
LGTM

**Pull request merged**